bedrock/base/templates/macros.html

```diff
@@ -268,7 +268,9 @@ <h3 class="mzp-c-sendyourself-title">
           <p class="mzp-c-sendyourself-field">
             <label for="{{ id }}-input" class="mzp-js-input-label" data-alt="{{ _('Enter your email or phone number') }}">{{ _('Enter your email') }}</label>
             <input id="{{ id }}-input" class="mzp-c-sendyourself-input" name="phone-or-email" type="text" required>
-            <button type="submit" class="mzp-c-button">{{ _('Send') }}</button>
+            <div class="form-submit">
```

Comment:
Is this change to the HTML needed for the test to pass? You might be able to use a selector like `.mzp-c-sendyourself-field button[type="submit"]` without changing the markup perhaps.